common/include/pcl/experimental/executor/default/inline_executor.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author: Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#pragma once
+
+#include <pcl/experimental/executor/default/base_executor.hpp>
```

Comment:
I would be happier if you used includes more atomically instead of using header bundles.